dbld/builddeps

```diff
@@ -222,7 +222,11 @@ function enable_dbgsyms {
 }
 
 function install_perf {
```

Comment:
I do not know the original intention behind `install_perf` (It looks like it wanted to handle package name differences between debian based distributions.), but currently we use the `packages.manifest` file for that.

My suggestion is to remove the `install_perf` function completely. (currently only the devshell.dockerfile calls it)

Reply:
The perf tools depend on the kernel versions available in those distributions, and I would rather not maintain them in packages.manifest, as the set of kernel versions supported in a distro might change, as distros regularly add extra kernels.

Reply:
As far as I see, `install_perf` was introduced with a157ec41493a147988049634c127538ec29b8feb in #1950.
I still don't know how could you use perf, as your host kernel might be totally different...

@szemere 
One does not simply move the content of `install_perf` into packages.manifest, as we have a logic here to select the correct package name and not a package name. Run this command on a debian/ubuntu distro: `apt-cache search --names-only "linux-tools.*generic"`.

@bazsi Can you clarify this, please? You introduced `perf` in `devshell` and I don't know someone else who is using it.

Reply:
Ooops, I didn't saw @bazsi's comment before writing mine.
From that I conclude that we should still add `perf` to devshell.